### ISSUE TITLE
remove departing BDFL from voting in every election

### ISSUE THREAD

**Issue by andrewhsu** [moby/tsc#20]
I propose the departing BDFL does not need a vote in every election.

From the [README.md](https://github.com/moby/tsc/blob/9e77cdc/README.md):
> The departing BDFL, Solomon Hykes, will receive one vote in every election.

**Comment by tianon** [comment]
I agree, his service was certainly appreciated (:heart:), but at this point I don't think this is still relevant.

**Comment by cpuguy83** [comment]
I also think it's fine to invite him to vote, no?
